eidas_node/proxy_service/views.py

```diff
@@ -101,6 +117,16 @@ def get_light_storage(self, backend: str, options: Dict[str, Any]) -> LightStora
         """
         return import_from_module(backend)(**options)
 
+    def get_auxiliary_storage(self, backend: str, options: Dict[str, Any]) -> AuxiliaryStorage:
```

Comment:
This can be a function in the module. It is defined twice and doesn't appear to need `self`.

Reply:
I'm thinking about where to move it. It will be used by both proxy service and connector views. Maybe `eidas_node.utils` or `eidas_node.storage.utils`? Should I move `get_light_storage` too?

Reply:
Yeah, I have noticed that too after commenting here :)

`eidas_node.storage.utils` sounds like a good place. `get_light_storage` could be moved as well (not necessarily now)

Reply:
`get_auxiliary_storage` moved.